inference-engine/src/mkldnn_plugin/nodes/mkldnn_embedding_segments_sum_node.cpp

```diff
@@ -36,13 +40,13 @@ MKLDNNEmbeddingSegmentsSumNode::MKLDNNEmbeddingSegmentsSumNode(const std::shared
     }
 
     std::string errPrefix = std::string("EmbeddingSegmentsSum layer with name '") + _layerName + "' ";
-    if (op->get_input_shape(INDICES_IDX).size() != 1)
+    if (op->get_input_partial_shape(INDICES_IDX).size() != 1)
```

Comment:
maybe better?
```suggestion
    if (op->get_input_partial_shape(INDICES_IDX).rank().get_length() != 1)
```

Reply:
was changed to `getInputShapeAtPort(INDICES_IDX).getRank() != 1ul`, thanks